README.md

```diff
@@ -64,13 +65,16 @@ client.trackTrace("trace message");
 ### Telemetry Processor
 
 ```javascript
-public addTelemetryProcessor(telemetryProcessor: (envelope: ContractsModule.Contracts.Envelope) => boolean)
+public addTelemetryProcessor(telemetryProcessor: (envelope: ContractsModule.Contracts.Envelope, context: { http.RequestOptions, http.ClientRequest, http.ClientResponse, userProperties }) => boolean)

```

Comment:
That context was passed to telemetry processor so customer can extract certain values from http request and associate them with the telemetry collected by default. 

Like if you want to get the header value and add it as a custom property to the `RequestTelemetry`. 

So I'd suggest to revert it back and extend with other context if needed